source/manual/email-notifications-how-they-work.html.md

```diff
@@ -9,68 +9,53 @@ parent: "/manual.html"
 
 ## High-level overview
 
-The purpose of the email notifications system is to inform users when
-content they are interested in is added to or changed on GOV.UK. Users
-indicate their interest by subscribing to receive updates for an area of
-interest, such as topic or government department.
+The purpose of the email notifications system is to inform users when content they are interested in is added to or changed on GOV.UK. Users can subscribe to receive updates for an area of interest, such as a topic, government department, or a set of search results. Currentl subscriptions to individual content items are not supported.
 
-The applications that comprise the email notifications system are:
+> **Update September 2019:** the email system also now supports highly customised subscriptions for users completing the [Brexit Checker][brexit-checker], which operates in isolation from the rest of GOV.UK, and specifies its own notifications. More information can be found in [the ADR][message-adr] for these changes.
```

Comment:
Nitpick, using a `>` for an aside doesn't seem very semantic. It ends up rendered in a blockquote element that is intended for extended quotations. It's a shame there isn't an aside markdown. 

But I wonder if this would be fine just as a paragraph.

Reply:
Hmm...since developers never do extended quotations of anything, I think there's some scope for redefinition there. However, I agree it doesn't need it, so removed.

Reply:
Yeah, I think it just risks being an accessibility gotcha if anyone was to navigate these docs with a screenreader. It crosses the line from a semantic usage to a presentational one.  I think never might be a misappropriation though. It'd be relevant for us to quote from a manual for instance. 

Thanks for fixing 👍

Reply:
Oh yes, that's a fair point.

> It'd be relevant for us to quote from a manual for instance.

I await that glorious day 😉.